barretenberg/cpp/src/barretenberg/vm/tests/AvmMini_execution.test.cpp

```diff
@@ -60,24 +61,28 @@ TEST_F(AvmMiniExecutionTests, basicAddReturn)
                                "00000000"; // ret size 0
 
     auto bytecode = hex_to_bytes(bytecode_hex);
-    auto instructions = Execution::parse(bytecode);
+    auto instructions = Deserialization::parse(bytecode);
 
     // 2 instructions
     EXPECT_EQ(instructions.size(), 2);
```

Comment:
Prefer `EXPECT_THAT(instructions, SizeIs(2));`. More specific matchers generate better error messages.

Reply:
Great idea. Unfortunately it seems unsupoorted by our build system right now.